pipenv/cli.py

```diff
@@ -1764,7 +1766,7 @@ def do_py(system=False):
 @click.option('--dev', '-d', is_flag=True, default=False, help="Install package(s) in [dev-packages].")
 @click.option('--three/--two', is_flag=True, default=None, help="Use Python 3/2 when creating virtualenv.")
 @click.option('--python', default=False, nargs=1, help="Specify which version of Python virtualenv should use.")
-@click.option('--system', is_flag=True, default=False, help="System pip management.")
+@click.option('--system', is_flag=True, default=False, help="System pip management.", envvar='PIPENV_USE_SYSTEM')
```

Comment:
we don't use envvar anywhere else -- I don't think it's inherently bad, but i think we should stick to the conventions established.

Reply:
kk, I wasn't sure so I'll nix it